airflow/utils/json.py

```diff
@@ -71,7 +75,7 @@ def default(self, o: Any) -> Any:
                     return data[DATA]
             return data
         except TypeError:
-            raise
+            return repr(o)
```

Comment:
Is this one related?

Reply:
I'm not sure that returning `repr(o)` whatever the o type is a good idea, but I'm trying to generalize this, where we cannot handle all the types especially the user custom classes, so instead of raising an exception, we can return the value as string since it's just to show it in the UI.
What do you think?

Reply:
Makes sense. We should probably mention this is the encoder’s docstring.